english/forest/pockets/pockets.md

```diff
@@ -0,0 +1,11 @@
+You empty your pants pockets on the grass in front of you. A pocket knife, a block of magnesium, a flare, a first aid kit and a machete. Now that's good news!
+
+You bandage your leg with supplies from the first aid kit and make a leg splint out of two nearby branches and some of the remaining bandages.
+
+You notice it's getting darker. In the distance, you hear something. Could... Could it be? Yes! It's a helicopter! This must be a rescue helicopter! If only there was a way to signal them..
+
+You light the flare and wave it over your head! Suddenly you realize, that under all this thick foliage, there is no way they are able to see it! What to do??!
+
+You think quickly. Of course! You throw the flare at the tree that holds your parachute. Immediately, the flare sets it ablaze! You hear the sounds of the helicopter's rotor blades approaching. It lands in a small nearby field. You're saved! As you limp towards the helicopter, and see stewardesses approach you with champagne and refreshments, you think to yourself:
```

Comment:
This line and quite a few others are above the 120 character limit. This will make tools like `git diff` difficult to use. Could you wrap all the long lines? You don't need to add additional blank lines, just press enter somewhere in the middle of each line. You may also be able to get your text editor to do this automatically, depending on which one you use.